content/en/ddsql_reference/ddsql_preview/tags.md

```diff
@@ -2,12 +2,22 @@
 title: Querying Tags in DDSQL
 aliases:
 - /dashboards/ddsql_editor/reference/tags/
+- /ddsql_editor/reference/tags/
+- /ddsql_editor/tags
+further_reading:
+- link: "/ddsql_reference/"
+  tag: "Documentation"
+  text: "DDSQL Reference"
```

Comment:
Same note as before about being confused over which version this guide is for, especially with this added link to the current syntax at the bottom of the page. If it applies to both, can we make that explicit?

Reply:
Good point, I'll remove the reference since it doesn't apply to both!